conda_smithy/cli.py

```diff
@@ -43,7 +43,7 @@ def create_git_repo(target, msg):
         # prevent this:
         # bash: line 1: ./ci_support/run_docker_build.sh: Permission denied
         # ./ci_support/run_docker_build.sh returned exit code 126
-        subprocess.check_call(['git', 'update-index' '--chmod=+x' 'ci_support/run_docker_build.sh'], cwd=target)
+        subprocess.check_call(['git', 'update-index', '--chmod=+x', 'ci_support/run_docker_build.sh'], cwd=target)
```

Comment:
:scream: 
That is what you get when 99.99% of the devs don't use Windows :wink: